source/iofs/README.md

```diff
@@ -0,0 +1,38 @@
+# iofs
+
+Driver with file system interface (`io/fs#FS`) supported from Go 1.16.
+
+This Driver cannot be used with Go versions 1.15 and below.
+
+## Usage
+
+Directory embedding example
+
+```go
+package main
+
+import (
+	"embed"
+	"log"
+
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	"github.com/golang-migrate/migrate/v4/source/iofs"
+)
+
+//go:embed migrations
+var fs embed.FS
+
+func main() {
+	d, err := iofs.WithInstance(fs, "migrations")
```

Comment:
```suggestion
	d, err := iofs.WithInstance(fs, ".")
```

If you accept [my other suggestion above](https://github.com/golang-migrate/migrate/pull/472/files#r523459865) this would need to change to `.`